src/common/gpio_motor.rs

```diff
@@ -314,10 +317,13 @@ where
     }
 }
 
-impl<B> Stoppable for PwmABMotor<B>
+impl<B> Actuator for PwmABMotor<B>
 where
     B: Board,
 {
+    fn is_moving(&mut self) -> anyhow::Result<bool> {
+        Ok(self.board.get_pwm_duty(self.pwm_pin) != 0.0)
```

Comment:
I discussed this a bit with @randhid about whether a stalled motor should return true for `is_moving`, and there isn't really any way to do so for a GPIO motor. Effectively, `is_moving` is being implemented as `is_powered` here. Rand's thought is that there should be some threshold, but as a separate item of work.

Reply:
wait, I might be able to add the threshold in this PR. @randhid do you have a take on what a good threshold should be, or is it something we'd have to make configurable?

Reply:
default of 5% for now - configurable if someone complains in the future

Reply:
or sets a min_power actually - we can leverage that if it exists for micro-rdk motors.